tox.ini

```
@@ -4,27 +4,29 @@ envlist =
     black
     flake8
     isort
-    py35-django111
-    py{36,37,38}-django{111, 22, 30, master}
+    py{36,37,38,39}-dj{22, 30, 31, master}
```

Obsolete spacing in test version definition

```suggestion
    py{36,37,38,39}-dj{22,30,31, master}
```